client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.controller.js

```diff
@@ -1820,4 +1813,12 @@ angular.module("managerApp")
     self.hasWindowsCompatibilityIssue = function(category) {
         return self.vmInEdition.image && self.vmInEdition.image.type === 'windows' && category === 'vps';
     };
+
+    self.showFlex = function (category) {
+        return _.includes(["balanced", "cpu", "ram"], category);
```

Comment:
Do you think we need to make these changes in the list view also?

Reply:
There are no notions of categories in the list view.

Reply:
Alright